bindings/nodejs/lib/types/client/output_builder_params/basic-output-params.ts

```diff
@@ -9,7 +9,7 @@ export interface BasicOutputBuilderParams {
     /**
      * If not provided, minimum storage deposit will be used
      */
-    amount?: string;
+    amount?: bigint | string;
```

Comment:
should this be hexencodedamount, or jsut string is fine?

Reply:
Here too it's decimal, not hex.